packages/alcotest/alcotest.0.8.4/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "Alcotest is a lightweight and colourful test framework."
+description: """
+Alcotest exposes simple interface to perform unit tests. It exposes
+a simple TESTABLE module type, a check function to assert test
+predicates and a run function to perform a list of unit -> unit
+test callbacks.
+
+Alcotest provides a quiet and colorful output where only faulty runs
+are fully displayed at the end of the run (with the full logs ready to
+inspect), with a simple (yet expressive) query language to select the
+tests to run."""
+maintainer: "thomas@gazagnaire.org"
+authors: "Thomas Gazagnaire"
+license: "ISC"
+homepage: "https://github.com/mirage/alcotest/"
+doc: "https://mirage.github.io/alcotest/"
+bug-reports: "https://github.com/mirage/alcotest/issues/"
+depends: [
+  "ocaml" {>= "4.02.3"}
+  "jbuilder" {build & >= "1.0+beta10"}
+  "fmt" {>= "0.8.0"}
+  "astring"
+  "result"
+  "cmdliner"
+  "uuidm"
+]
+flags: light-uninstall
+build: [
+  ["jbuilder" "subst" "-n" name] {pinned}
```

Comment:
is `-n` correct?

Reply:
this is probably not needed indeed; not sure about the `liigh-uninstal` flag too.

Reply:
```suggestion
  ["jbuilder" "subst" ] {pinned}
```

Reply:
The light-uninstall shouldn't be needed with jbuilder or dune.
But I was more thinking about this PR https://github.com/ocaml/opam-repository/pull/12279. Do we not need `"-p" name" ?

Reply:
Ha you are right, it's still using jbuilder!